MAUI/Cartesian-Charts/neumorphicUI-column-chart.md

```diff
@@ -0,0 +1,245 @@
+---
+layout: post
+title: Neumorphic UI Column Chart in .NET MAUI Chart control | Syncfusion
+description: Learn here about how to create the Neumorphic UI chart types using Syncfusion .NET MAUI Chart (SfCartesianChart) control.
+platform: maui
+control: SfCartesianChart
+documentation: ug
+---
+
+# Neumorphic UI Chart using Syncfusion .NET MAUI Column Chart
+
+Learn to create visually stunning Neumorphic charts in using Syncfusion [.NET MAUI Column Chart](). Follow a step-by-step tutorial utilizing a custom class that inherits from `IDrawable` and `Graphics view` for a seamless blend of design and data representation in your application.
+
+
+## Step 1: Implementing SfShadowDrawer Class
+
+The `SfShadowDrawer` class is a base class that implements the `IDrawable` interface. It serves as a foundation for creating drawable elements with shadow effects. This class provides properties and methods to control shadow properties, such as offset, blur, color, and opacity.
+
+## Step 2: Implementing Draw Method
+
+Implements the `IDrawable` interface method `Draw`. This method is responsible for drawing the shadow by applying the shadow effect and filling the rounded rectangle.
```

Comment:
@DhanarajRajendran 
Looks like sentence not having proper information & incomplete, modify as
Implement the Draw method for the IDrawable interface in the custom SfShadowDrawer class.